### PR TITLE
WFLY-8588 Tests in the same module should not use different base pack…

### PR DESCRIPTION
**Pull request by stuartwdouglas**
…ages

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
Retest this please

**Review by ctomc — COMMENTED** [review]
elytron tests are new and don't clash with other stuff. they should be kept in the new package.

**Comment by kabir** [comment]
Retest this please

**Comment by jamezp** [comment]
I tried pulling this down and rebasing to test an Arquillian issue and it seems to have some issues. There are several compile errors it seems.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/VirtualHostTestCase.java [hunk 1]
```diff
@@ -17,10 +17,10 @@
  * You should have received a copy of the GNU Lesser General Public
  * License along with this software; if not, write to the Free
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
- * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ * 2110-1301 USA, or see the FSF site: http://www.fsf.org.
```
Comment (darranl) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/VirtualHostTestCase.java]: Is it correct to drop the leading 0 from the phone number?

**Comment by darranl** [comment]
Maybe time to revisit and instead of renaming the tests split them into their own module.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/VirtualHostTestCase.java (hunk 1 above)
Comment (jamezp) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/VirtualHostTestCase.java]: That's the postal code so it should probably not be changed.

Comment (darranl) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/VirtualHostTestCase.java]: LOL yes ;-)

**Comment by stuartwdouglas** [comment]
Closing for now, but I want to discuss test suite changes in Brno

**Comment by darranl** [comment]
+1 We do need to revisit the testsuite structure again, the current changes don't really reflect our last discussion.

**Pull request closed without merging**